test_runner/fixtures/neon_fixtures.py

```diff
@@ -3796,8 +3796,7 @@ def create(
             pageserver_id=pageserver_id,
             allow_multiple=allow_multiple,
         )
-        path = Path("endpoints") / self.endpoint_id / "pgdata"
-        self.pgdata_dir = os.path.join(self.env.repo_dir, path)
+        self.pgdata_dir = self.pg_data_dir_path()
```

Comment:
@bayandin I'm not yet familiar enough with regress test suite, so I wonder how far are we from getting this PR merged? Am I right that the main problem now is that sometimes we get file already exists here?

That'd help testing the https://github.com/neondatabase/neon/pull/8575